Runtimes/Core/cmake/modules/Plist.cmake

```diff
@@ -0,0 +1,32 @@
+function(generate_plist project_name project_version target)
+  if(NOT CMAKE_SYSTEM_NAME STREQUAL "Darwin")
+    return()
+  endif()
+  set(PLIST_INFO_PLIST "Info.plist" CACHE STRING "Plist name")
```

Comment:
This doesn't really help explain the cache variable. I think that we might as well as elide the description or should rework it to be more helpful.